include/classes/DAO/DAO.class.php

```diff
@@ -63,22 +60,33 @@ function DAO()
 	function execute($sql)
 	{
 		$sql = trim($sql);
-		$result = mysql_query($sql, $this->db) or die($sql . "<br />". mysql_error());
+		$result = mysqli_query($this->db, $sql) or die($sql . "<br />". mysqli_error($this->db));
 
 		// Deal with "select" statement: return false if no row is returned, otherwise, return an array
 		if ($result !== true && $result !== false) {
 			$rows = false;
 			
-			while ($row = mysql_fetch_assoc($result)){
+			while ($row = mysqli_fetch_assoc($result)){
 				if (!$rows) $rows = array();
 				
 			    $rows[] = $row;
 			}
-			mysql_free_result($result);
+			mysqli_free_result($result);
 			return $rows;
 		}
 		return true;
 	}
 
+	function addSlashes($string){
```

Comment:
It would be nice to follow the same coding convention in this file to have the opening curly bracket (`{`) in a new line. 

The same comment applies across other files. Thanks.